src/assets/javascripts/angular/controllers/widgets/finaid/finaidSummaryController.js

```diff
@@ -21,6 +24,18 @@ angular.module('calcentral.controllers').controller('FinaidSummaryController', f
     updateUrl();
   };
 
+  var showTipGiftAid = function() {
```

Comment:
@marcpechaitis - Sorry to be picky, but shouldn't these be named something like `toggleTipGiftAid` since it goes back and forth?

Reply:
@developish I can't disagree with that... the reason I was using `showTipXYZ` name is b/c I saw that somewhere else in the codebase